man/systemd-vmspawn.xml

```diff
@@ -135,25 +140,20 @@
         <varlistentry>
           <term><option>--vsock=<replaceable>BOOL</replaceable></option></term>
 
-          <listitem>
-            <para>Configure whether to use VSOCK networking.</para>
+          <listitem><para>If <option>--vsock=</option> is not specified VSOCK networking support will be
+          detected automatically. If true, VSOCK networking is always used, and if false, VSOCK networking is never used.</para>
 
-            <para>If the option is not specified VSOCK support will be detected automatically.  If yes is
-            specified VSOCK is always used, and vice versa if no is set VSOCK are never used.</para>
-            <xi:include href="version-info.xml" xpointer="v255"/>
-          </listitem>
+          <xi:include href="version-info.xml" xpointer="v255"/></listitem>
         </varlistentry>
 
         <varlistentry>
           <term><option>--vsock-cid=<replaceable>CID</replaceable></option></term>
 
           <listitem>
-            <para>Configure vmspawn to use a specific CID for the guest.</para>
-
-            <para>If the option is not specified or an empty argument is supplied the guest will be assigned a random CID.</para>
-
-            <para>Valid CIDs are in the range <constant>3</constant> to <constant>4294967294</constant> (<constant>0xFFFF_FFFE</constant>).
-            CIDs outside of this range are reserved.</para>
+            <para>Sets the specific CID to use for the guest.
+            Valid CIDs are in the range <constant>3</constant> to <constant>4294967294</constant> (<constant>0xFFFF_FFFE</constant>).
+            CIDs outside of this range are reserved. By default vmspawn will attempt to derive a CID for the guest based on the machine name,
```

Comment:
```suggestion
            CIDs outside of this range are reserved. By default vmspawn will attempt to derive a CID for the guest derived from the machine name,
```

Based on to me implies the machine name is part of the value, "derived from" more precisely communicates some complicated hashing may be involved. "derived from factors including the machine name" might be more appropriate depending on how much detail it is important to communicate.